src/components/Flight/FlightItinerarySummary.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FlightLeg } from '../../schema/flight.schema';
+
+interface FlightItinerarySummaryProps {
+  data: FlightLeg[];
+}
+const FlightItinerarySummary = ({ data }: FlightItinerarySummaryProps) => {
+  return (
+    <div>
+      {data.map((leg, index) => {
+        return (
+          <>
+            <p>
```

Comment:
You'll have to add a key in here somewhere so we don't keep getting this error in the console:
![Screenshot 2022-08-21 141420](https://user-images.githubusercontent.com/84279128/185805066-64320cb1-a807-41df-8200-2897fd50b18f.png)

Reply:
added keys for the p tags